runtime-extension-model/src/test/java/org/mule/runtime/config/internal/validation/MuleSDKOperationDoesNotHaveDeprecatedExpressionTestCase.java

```diff
@@ -0,0 +1,250 @@
+  }
+
+  @Test
+  public void withoutExpression() {
+    when(expressionLanguage.collectScopePhaseValidationMessages(contains("Mule::lookup("), anyString(), any()))
+        .thenReturn(new TestScopePhaseValidationMessages(singletonList(new TestScopePhaseValidationItem("2.5.0",
+                                                                                                        "Mule::lookup"))));
+    final Optional<ValidationResultItem> msg = runValidation(XML_NAMESPACE_DEF +
+        "<operation:def name=\"someOp\"><operation:body>" +
+        "<set-payload value=\"10\"/>" +
+        "</operation:body></operation:def>" +
+        XML_CLOSE).stream().findFirst();
+    assertThat(msg, is(empty()));
+  }
+
+  @Test
+  public void withMultipleCorrectExpressions() {
+    when(expressionLanguage.collectScopePhaseValidationMessages(contains("keySet("), anyString(), any()))
+        .thenReturn(new TestScopePhaseValidationMessages(singletonList(new TestScopePhaseValidationItem("2.3",
+                                                                                                        "dw::core::Objects::keySet"))));
+    when(expressionLanguage.collectScopePhaseValidationMessages(not(contains("keySet(")), anyString(), any()))
+        .thenReturn(new TestScopePhaseValidationMessages());
+    final Optional<ValidationResultItem> msg = runValidation(XML_NAMESPACE_DEF +
+        "<operation:def name=\"someOp\"><operation:body>" +
+        "<set-payload value=\"10\"/>" +
+        "<logger level=\"WARN\" message=\"#[payload ++ 10]\"/>" +
+        "<set-payload value=\"#[payload + 1]\"/>" +
+        "<set-variable value=\"#[someFunction(10)]\"/>" +
+        "</operation:body></operation:def>" +
+        XML_CLOSE).stream().findFirst();
+    assertThat(msg, is(empty()));
+  }
+
+  @Test
+  public void withOneOverManyExpressionsDeprecated() {
+    when(expressionLanguage.collectScopePhaseValidationMessages(anyString(), anyString(), any()))
+        .thenReturn(new TestScopePhaseValidationMessages());
+    when(expressionLanguage.collectScopePhaseValidationMessages(contains("Mule::lookup("), anyString(), any()))
+        .thenReturn(new TestScopePhaseValidationMessages(singletonList(new TestScopePhaseValidationItem("2.5.0",
+                                                                                                        "Mule::lookup"))));
+    final Optional<ValidationResultItem> msg = runValidation(XML_NAMESPACE_DEF +
+        "<operation:def name=\"someOp\"><operation:body>" +
+        "<set-payload value=\"10\"/>" +
+        "<logger level=\"WARN\" message=\"#[payload ++ 10]\"/>" +
+        "<set-payload value=\"#[payload + 1]\"/>" +
+        "<logger level=\"WARN\" message=\"#[%dw 2.0 --- { 'v' : Mule::lookup('someFlow') }]\"/>" +
+        "<set-variable value=\"#[someFunction(10)]\"/>" +
+        "</operation:body></operation:def>" +
+        XML_CLOSE).stream().findFirst();
+    assertThat(msg, is(CoreMatchers.not(empty())));
```

Comment:
static import

Reply:
I am already using another "not" (AdditionalMatchers)